drivers/web/framework/httprequest/session.py

```diff
@@ -3,7 +3,7 @@
 
 from drivers.web.framework.http_response import template_http_response
 from drivers.web.framework.httprequest.http_request import HttpRequest
-
+from drivers.web.application.settings import AUTH_REDIRECT
```

Comment:
framework cannot import from application. That relationship is exactly what we want to remove by not using index.html, right? You must use the same strategy to inject TEMPLATES (i.e call a configure on config)

Reply:
should i create a configure called like "configure_auth_redirect" receinving a template name? If yes, how should i save this data to use on session?

Reply:
Think i got it. I've made "configure_auth_redirect" that receives the auth_service template name and storage it on a new variable from template called "auth_redirect_template", that starts with an empty string. Is it correct?

Reply:
In that way, "auth_needed" from session now uses a template variable and stop interacting with application.

Reply:
![image](https://github.com/SistemaFinanceiroNacional/login-senha-ryan/assets/95293822/55c256c6-e35f-4b28-93e6-3cfbb666830b)
That's not working correctly. Got "jinja2.exceptions.TemplateNotFound" because the variable does not save the value.